java/src/jni/exceptionImp.c

```diff
@@ -318,11 +331,25 @@ h5badArgument(JNIEnv *env, const char *functName)
  *  exception.
  */
 jboolean
-h5unimplemented(JNIEnv *env, const char *functName)
+h5unimplemented(JNIEnv *env, const char *message)
 {
-    return H5JNIErrorClass(env, functName, "java/lang/UnsupportedOperationException");
+    return H5JNIErrorClass(env, message, "java/lang/UnsupportedOperationException");
 } /* end h5unimplemented() */
 
+/*
+ *  indicate that assert statement fails
+ *  Create and throw an 'AssertionError'
+ *
+ *  Note:  This routine never returns from the 'throw',
+ *  and the Java native method immediately raises the
+ *  exception.
+ */
+jboolean
+h5assertion(JNIEnv *env, const char *message)
```

Comment:
Unless github is showing the diff strangely, it looks the the implementation for this was added twice? Once above as well as this one.

Reply:
ARRRGH!
My computer is running like 5 builds and I end up with a mess when editing!